apps/mark-scan/backend/src/custom-paper-handler/state_machine.test.ts

```diff
@@ -283,6 +296,45 @@ test('voting flow happy path', async () => {
   await waitForStatus('not_accepting_paper');
 });
 
+describe('removing ballot during presentation state', () => {
+  test('is a no op if USE_MOCK_PAPER_HANDLER=true', async () => {
```

Comment:
Need this to no-op because paper handler status always reports `NO_PAPER_ANYWHERE` when using mock paper handler.